package.json

```diff
@@ -43,6 +43,11 @@
     "inline-styles",
     "react"
   ],
+  "files": [
+    "package.json",
+    "lib",
```

Comment:
You don't need to build the lib folder, that's done before each release so it's on npm. See https://unpkg.com/polished@1.0.0/lib/